### DISCUSSION TITLE
It'd be nice to be able to set the _Override User-Initiated Action Lifespan_ values over the API

### DISCUSSION THREAD

**Discussion by vilmosnagy** [keycloak/keycloak#8844] [Q&A]
### Description

I'd like to be able to set the _Realm settings/Tokens/Override User-Initiated Action Lifespan_ values over the API.

### Motivation

It'd be nice to be able to manage these values from terraform, and if I understand correctly,KeyCloak does not provide an API endpoint to do this.

**Comment by vilmosnagy** [comment]
If I understand KeyCloak correctly, I should:
 - add the `Map<String, Integer> actionTokenGeneratedByUserLifespanOverrides` field to the `RealmRepresentation` class
 - implement the mapping in the `RepresentationToModel` and in the `ModelToRepresentation` classes

Would you merge a PR with these changes?

**Comment by stianst** [comment]
As you can set these through the admin console it is already possible to do this through the Admin APIs. So I would suggest looking at the API docs, and also on what requests the admin console sends when these are being updated there.

**Reply by vilmosnagy** [reply]
@stianst am I correctly assuming that I should put these into the `RealmRepresentation.attributes` map as `actionTokenGeneratedByUserLifespan.<token-id>: <validity-in-seconds>`?

Thanks,
